settings.gradle

```diff
@@ -1,5 +1,6 @@
 include ':sqlbrite'
 include ':sqlbrite-kotlin'
 include ':sample'
+include ':sqlbrite-lint'
```

Comment:
super nit: can you move this up so it's properly alphabetized with the other sqlbrite-prefixed modules. "sample" is intentionally last